src/components/generic/Badge/Badge.scss

```diff
@@ -0,0 +1,11 @@
+.wxc-badge {
+  padding: .5rem 1rem .4375rem 1.0625rem;
+  font-size: 1rem;
+  color: var(--wxc-badge-text-color);
+  background: var(--wxc-badge-background);
+  border: var(--wxc-badge-border);
+  box-shadow: 0 .5rem 1rem #{$wxc-black-alpha-16}, 0 0 .06rem #{$wxc-black-alpha-14};

```

Comment:
we should not have actual colors in the components' css, please add the shadow in a variable, eg --wxc-badge--box-shadow

check if the shadows are always the same for all components, in which case you could use a generic --wxc-box-shadow instead, for all shadows in the components

Reply:
there are different type of box-shadows